src/tools/react-native.ts

```diff
@@ -327,27 +327,19 @@ export function updateExpoRouterSrcDir(toolbox: GluegunToolbox) {
   })
 }
 
-export async function cleanupExpoRouterConversion(toolbox: GluegunToolbox) {
-  const { system, parameters, print } = toolbox
-
-  // debug?
-  const debug = boolFlag(parameters.options.debug)
-  const log = <T = unknown>(m: T): T => {
-    debug && print.info(` ${m}`)
-    return m
-  }
+export function cleanupExpoRouterConversion(toolbox: GluegunToolbox, targetPath: string) {
+  const { filesystem } = toolbox
 
-  await system.run(
-    log(`
-      \\rm src/app.tsx
-      mkdir src/components/ErrorBoundary
-      mv src/screens/ErrorScreen/* src/components/ErrorBoundary
-      rm App.tsx
-      rm ignite/templates/screen/NAMEScreen.tsx.ejs
-      rm -rf ignite/templates/navigator
-      rm -rf src/screens
-      rm -rf src/navigators
-      rm -rf app
-    `),
+  const workingDir = filesystem.cwd(targetPath)
+  workingDir.cwd("src").remove("app.tsx")
```

Comment:
It seems that commands aren't quite working on my Windows machines, I'm stilling getting `src/app.tsx` and some of the like:
![image](https://github.com/user-attachments/assets/faa3b5d2-8689-41dd-9398-91092a62fb72)